src/cli/pack_command.zig

```diff
@@ -1359,7 +1359,9 @@ pub const PackCommand = struct {
                         var files_array = _files_array;
                         while (files_array.next()) |files_entry| {
                             if (files_entry.asString(ctx.allocator)) |file_entry_str| {
-                                const parsed = try Pattern.fromUTF8(ctx.allocator, file_entry_str) orelse continue;
+                                var path_buf: PathBuffer = undefined;
```

Comment:
let's move the `path_buf` to right before the `while` loop, and reuse it for each pattern.
```suggestion
                                var path_buf: PathBuffer = undefined;
                                while (...) |files_entry| {
                                    ....
                                }
```